src/Api/Utilities/EnumMatchesAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Utilities;
+
+public class EnumMatchesAttribute<T>(params T[] accepted) : ValidationAttribute
```

Comment:
I added this attribute to validate an incoming `PlanType`, but it can be used for any enum.